doc/nrf/ug_matter_gs_adding_clusters.rst

```diff
@@ -182,30 +182,46 @@ In each iteration, an event is dequeued and a corresponding event handler is cal
 Add new events
 --------------
 
-After you copied the template sample application, the following events are present in the :c:struct:`AppEvent` structure in the :file:`src/app_event.h` file, each representing one action of the factory reset functionality:
+The possible events which can be handled by Matter samples are defined in :file:`samples/matter/common/src/event_types.h` in :c:enum:`AppEventType` enumeration.
+After you copied the template sample application, the following events are used in the application, each representing one action of the factory reset functionality:
 
-* :c:enum:`FunctionPress` - For pressing **Button 1**.
-* :c:enum:`FunctionRelease` - For releasing **Button 1**.
-* :c:enum:`FunctionTimer` - For factory reset timeout.
+* :c:enum:`ButtonPushed` - For pressing **Button 1**.
+* :c:enum:`ButtonReleased` - For releasing **Button 1**.
+* :c:enum:`Timer` - For factory reset timeout.
 
-To model the behavior of the sensor, add the following new :c:enum:`EventType` events:
+To model the behavior of the sensor, add the following new :c:enum:`AppEventType` events:
 
 * :c:enum:`SensorActivate` - For sensor activation.
 * :c:enum:`SensorDeactivate` - For sensor deactivation.
 * :c:enum:`SensorMeasure` - For sensor measurement update.
 
-For example, the edited :c:enum:`EventType` can look as follows:
+For example, the edited :c:enum:`AppEventType` can look as follows:
 
 .. code-block:: C++
 
-   enum EventType : uint8_t { FunctionPress, FunctionRelease, FunctionTimer, SensorActivate, SensorDeactivate, SensorMeasure };
-   enum UpdateLedStateEventType : uint8_t { UpdateLedState = SensorMeasure + 1 };
+   enum class AppEventType {
```

Comment:
I don't think someone needs to add events like StartSMPAdvertising or Lock if those will not be supported anyway. I would try to mention only necessary events.

Reply:
Note that to avoid a mess I have moved the `AppEventType` enum to a separate file, so it contains all possible events accessible for all samples. I believe it does not affect the flash usage much. Here I have included the whole enumeration to indicate how it would look overall (similar was done before my changes). I may just strip all except 3 last ones and use a comment like "Original content" (as for Init() method).

Reply:
Discussed F2F

Reply:
Done